flake8-requirements.txt

```diff
@@ -1,27 +1,27 @@
-flake8~=3.8.3
+flake8==3.8.3
 
 # Plugins
 # Ref: https://github.com/DmytroLitvinov/awesome-flake8-extensions
 
-flake8-bandit~=2.1.2                    # runs bandit
```

Comment:
Don't freeze bandit please, we want this to always pull in the newest version regardless of whether dependabot has updated it or not.